python_ta/reporters/templates/stylesheet.css

```diff
@@ -7,12 +7,12 @@ body {
 }
 
 header {
-    padding: 20px;
-	font-size: 14px;
+    padding: 30px;
     font-weight: normal;
     -webkit-font-smoothing: antialiased;
     color: #fff;
-    text-align: center;
+    margin-left: 20px;
```

Comment:
Instead of setting margins on this element, set padding on the containing `body` element.